lib/ansible/module_utils/network/vyos/utils/utils.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+# Copyright 2019 Red Hat
+# GNU General Public License v3.0+
+# (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+# utils
+
+
+def search_obj_in_list(name, lst, key='name'):
```

Comment:
If this function is used across platforms for resource modules can it be moved to `network/common/utils`